src/xwayland-unmanaged.c

```diff
@@ -79,6 +82,11 @@ unmanaged_handle_unmap(struct wl_listener *listener, void *data)
 			return;
 		}
 	}
+	/*
+	 * Mark the node as gone so a racing configure event
+	 * won't try to reposition the node while unmapped.
+	 */
+	unmanaged->node = NULL;
```

Comment:
This should probably be moved up before any of the `return;` paths.  Perhaps right near the two `wl_list_remove()` calls.

Reply:
This then LGTM

Reply:
> This should probably be moved up before any of the return; paths. Perhaps right near the two wl_list_remove() calls.

Should really have seen that, updated. Thanks for the review!